tests/trainer/test_fsdp_checkpoint.py

```diff
@@ -556,7 +556,8 @@ def test_fsdp_partitioned_state_dict_load(world_size, tmp_path: pathlib.Path, st
 @pytest.mark.parametrize('autoresume', [False, True])  # True commented out for now
 @pytest.mark.parametrize('num_shards', [2, 4, 7])
 @pytest.mark.parametrize('sharding_strategy', ['FULL_SHARD', 'SHARD_GRAD_OP'])
-@pytest.mark.skipif(version.parse(torch.__version__) < version.parse('2.0.1'), reason='requires PyTorch 2.01 or higher')
+@pytest.mark.skipif(version.parse(torch.__version__) < version.parse('2.0.1'),
```

Comment:
should this be <=?

Reply:
Nope, it is for 2.0. I just fixed a typo